ntbs-service/Pages/Notifications/LegacyView.cshtml

```diff
@@ -0,0 +1,56 @@
+﻿@page "/Notifications/LegacyView/{LegacyNotificationId}"
+
+@using ntbs_service.Helpers
+@model ntbs_service.Pages.Notifications.LegacyView
+
+@{
+    var classSuffix = CssClassHelpers.GetClassSuffixForNotificationStatus(Model.NotificationBanner.NotificationStatus);
+    var notificationStatusTriangleCssClass = $"notification-banner-triangle notification-banner-triangle{classSuffix}";
+}
+
+<nhs-width-container container-width="Standard">
+
+    <nhs-grid-row>
+        <nhs-grid-column grid-column-width="Full">
+            <div class="notification-banner-container">
+                <partial name="../Shared/_NotificationBanner.cshtml" model="@Model.NotificationBanner" />
+                <div class="@notificationStatusTriangleCssClass"> </div>
+            </div>
+        </nhs-grid-column>
+    </nhs-grid-row>
+
+@if (Model.LegacyImportResult == null)
+{
+    <div class="ondemand-migration-description-container">
+        You must import this notification to view the full details of the record.
+        <br/>
+        Import may take a few moments.
+        <br/>
+        Do you wish to proceed?
+        <br/>
+        <form method="post">
```

Comment:
Looks like the page could use some extra spacing between the button and the rest of the text, looks a little squished atm

Reply:
Possibly also some spacing between the banner and the text too :)